save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/WarnPlugin.kt

```diff
@@ -80,16 +94,37 @@ class WarnPlugin(testConfig: TestConfig) : Plugin(testConfig) {
         )
     }
 
+    /**
+     * @param path
+     * @return name of the temporary file
+     */
+    internal fun createTestFile(path: Path): String {
+        val tmpDir = (FileSystem.SYSTEM_TEMPORARY_DIRECTORY / WarnPlugin::class.simpleName!!)
+
+        if (fs.exists(tmpDir) && atomicInt.get() == 0) {
+            fs.deleteRecursively(tmpDir)
+            fs.createDirectory(tmpDir)
+        } else if (!fs.exists(tmpDir)) {
+            fs.createDirectory(tmpDir)
+        }
+
+        val fileName = testFileName()
+        fs.write(fs.createFile(tmpDir / fileName)) {
+            fs.readLines(path).forEach {
+                if (!it.contains(";warn:")) {
```

Comment:
```suggestion
                if (!warningsInputPattern.matches(it)) {
```
We have a configuration option for this. Also, please don't force-push after reviews.

Reply:
Force-push must be done after the merge the latest changes from _main_

Reply:
Use `merge from main` instead of `rebase`